### ISSUE TITLE
AZ function Dotnet app with eventhub trigger: 404 message entity could not be found

### ISSUE THREAD

**Issue by marchenm** [kedacore/keda#773]
deployment:
```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: demoapp
  namespace: demo-ns
  labels:
    app: demoapp
data:
  DemoAppEventHubName: "demoapp"
  DemoAppConsumerGroupName: "demoappconsumer"
  EventhubConnectionString: "Endpoint=sb://demo-app-eventhub.servicebus.windows.net/;SharedAccessKeyName=demokey;SharedAccessKey=adasfsdafdasdfasdfasdfasdfasd=;TransportType=AmqpWebSockets"
  AzureWebJobsStorage: "DefaultEndpointsProtocol=https;AccountName=demoappsa;AccountKey=asdfadfasdfasdfasdf==;EndpointSuffix=core.windows.net"
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: demoapp
  namespace: demo-ns
  labels:
    app: demoapp
spec:
  selector:
    matchLabels:
      app: demoapp
  template:
    metadata:
      labels:
        app: demoapp
    spec:
      containers:
      - name: demoapp
        image: demoapp
        envFrom:
        - configMapRef:
            name: demoapp
---
apiVersion: keda.k8s.io/v1alpha1
kind: ScaledObject
metadata:
  name: demoapp
  namespace: demo-ns
  labels:
    deploymentName: demoapp
spec:
  scaleTargetRef:
    deploymentName: demoapp
  triggers:
  - type: azure-eventhub
    metadata:
      type: eventHubTrigger
      connection: EventhubConnectionString
      storageConnection: AzureWebJobsStorage
      blobContainer: 'azure-webjobs-eventhub'
      consumerGroup: '%DemoAppConsumerGroupName%'
      eventHubName: '%DemoAppEventHubName%'
---
```
Keda logs:
```
"msg":"unable to get runtimeInfo for isActive","error":"unhandled error link a0b394d6-215a-4f54-9772-20bc57128cf9: status code 404 and description: The messaging entity 'sb://demo-app-eventhub.servicebus.windows.net/' could not be found. To know more visit https://aka.ms/sbResourceMgrExceptions. ","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128\ngithub.com/kedacore/keda/pkg/scalers.(*AzureEventHubScaler).IsActive\n\tkeda/pkg/scalers/azure_eventhub_scaler.go:152\ngithub.com/kedacore/keda/pkg/handler.(*ScaleHandler).handleScaleDeployment\n\tkeda/pkg/handler/scale_loop.go:119\ngithub.com/kedacore/keda/pkg/handler.(*ScaleHandler).handleScale\n\tkeda/pkg/handler/scale_loop.go:45\ngithub.com/kedacore/keda/pkg/handler.(*ScaleHandler).HandleScaleLoop\n\tkeda/pkg/handler/scale_loop.go:28"
```
Not sure of where the issue is. when I run the app without the scaler it works.

**Comment by tomkerkhove** [comment]
@marchenm Is there anything we can improve on our end or did you manage to figure it out?

**Comment by marchenm** [comment]
@tomkerkhove I managed to figure it out. 

My connection string didn't have the `EntityPath` appended to it. From the docs I wasn't sure if the scaler would append based on the `eventHubName` passed in. 

Also I had to change the `consumerGroup` to the actual consumer group. Setting it to the env var used by the deployment didn't work for me. 

Also the scaler that the func tools spit out wasn't usable, but thats an issue for that repository.
